internal/learn/iota-token/staking-iota.md

```diff
@@ -0,0 +1,99 @@
+- the ASMB distribution during the second staking period will happen on a **daily fixed rate**
+- assuming 14 Billion ASMB tokens will be available to distribute, this would lead to a daily payout of 
+    - 21.909.233 ASMB tokens to the IOTA stakers over 639 days
+    - the amount of individual received ASMB tokens per day depends on the total percentual part an IOTA staker holds of the total daily staked amount of IOTA tokens.
+    - the fewer total IOTA tokens are staked, the higher the individual payout of theis 21.909.233 ASMB tokens per day for an IOTA staker.
+
+![assembly token release](/img/learn/iota-token/assembly-token-release.png)
+
+## 3.) Stake Assembly tokens to secure and validate the Assembly network
+
+Assembly will have two different ways to reward Assembly token stakers.
+1. Reward Assembly stakers for holding ASMB tokens and distributing newly minted ASMB tokens as part of the inflationary token distribution.
+    - The yearly inflation of the Assembly supply is set to 8%. This inflation rate is planned to be reduced over the years by community governance decisions.
+    - The final mechanism of how this distribution will happen is not yet published
+
+2. Reward Assembly stakers to secure the Assembly chains by staking their ASMB tokens with a Validator.
+    - This mechanism is crucial to the security and functionality of the Assembly smart contract chains. Validators in the IOTA smart contract chains deployed in Assembly need to deposit Assembly tokens as a security bond in the Assembly Root chain contract. This stake deposited by validators decides in which chains they can participate as validators. The higher the stake a validator can offer as a security deposit, the more likely this validator will be chosen to secure high-value smart contract chains. Those chains will likely be more profitable for the validator, so the incentive to deposit more stakes as a security deposit in Assembly is very high. You can read details of that mechanism [here.](https://blog.assembly.sc/meet-assembly-2-validators-and-committees/)
+    - A Validator must deposit some amount of tokens that he owns himself and can also deposit tokens that Assembly tokens holders deposited to the validator.
+    - For this delegated stake from usual Assembly holders, the validators will most likely be willing to pay a part of the gained profits from validating smart contracts to the Assembly holders that deposited their tokens to the validator.
+    - This involves a potential risk, as a deposited stake in the Assembly root chain contract can be partially or fully slashed if the validator acts maliciously. But, besides this small theoretical risk, delegating stake to a validator will also offer higher rewards for Assembly stakers, as a share of the validators profit will be distributed to the stakers that delegate the stake to the validator.
```

Comment:
This part needs some more love and clear wording, because it is quite important, but I don't feel like it gets the point across.